README.md

```diff
@@ -1,46 +1,30 @@
-[hyper's `Client` type](https://docs.rs/hyper/latest/hyper/client/struct.Client.html)
-providing a simpler async interface as well as a blocking interface. For a more
-feature-rich HTTP client use [`reqwest`](https://crates.io/crates/reqwest) instead.
-
-**Example use:**
-
-```rust
-use simple_hyper_client::{Client, HttpConnector};
-let connector = HttpConnector::new();
-let client = Client::with_connector(connector);
-let response = client.get("http://example.com/")?.send().await?;
-```
-
-Using blocking client:
-```rust
-use simple_hyper_client::{blocking::Client, HttpConnector};
-let connector = HttpConnector::new();
-let client = Client::with_connector(connector);
-let response = client.get("http://example.com/")?.send()?;
-```
-
-There is a TLS connector available as well which can be enabled using the
-`native-tls` feature of this crate (not enabled by default):
-
-```rust
-use native_tls::TlsConnector;
-use simple_hyper_client::{Client, HttpsConnector};
-let tls = TlsConnector::new();
-let connector = HttpsConnector::new(tls.into());
-let client = Client::with_connector(connector);
-let response = client.get("https://example.com/")?.send().await?;
-```
-
-
-# Contributing
+This workspace provides crates:
+
+- [`simple-hyper-client`](./simple-hyper-client/):a wrapper for [hyper's `Client` type](https://docs.rs/hyper/latest/hyper/client/struct.Client.html) providing a simpler async interface as well as a blocking interface. For a more feature-rich HTTP client use [`reqwest`](https://crates.io/crates/reqwest) instead.
+- [`simple-hyper-client-native-tls`](./simple-hyper-client-native-tls/): Provides [`NetworkConnector`](https://docs.rs/simple-hyper-client/latest/simple_hyper_client/trait.NetworkConnector.html) implementation using [tokio-native-tls](https://crates.io/crates/tokio-native-tls).
+- [`simple-hyper-client-rustls`](./simple-hyper-client-rustls/): Provides [`NetworkConnector`](https://docs.rs/simple-hyper-client/latest/simple_hyper_client/trait.NetworkConnector.html) implementation using [tokio-rustls](https://crates.io/crates/tokio-rustls).
+
+## Example
+
+Please check crate [`example`](./example/):
+
+- Async client: [example/src/bin/async_client.rs](./example/src/bin/async_client.rs)
+  - Use native-tls: `cargo run -p example --bin async_client --features native-tls`
+  - Use rustls: `cargo run -p example --bin async_client --features rustls`
+- Blocking client: [example/src/bin/client.rs](./example/src/bin/client.rs)
+  - Use native-tls: `cargo run -p example --bin client --features native-tls`
+  - Use rustls: `cargo run -p example --bin client --features rustls`
+
+## Contributing
 
 We gratefully accept bug reports and contributions from the community.
 By participating in this community, you agree to abide by [Code of Conduct](./CODE_OF_CONDUCT.md).
 All contributions are covered under the Developer's Certificate of Origin (DCO).
 
+For guide of making contribution please check [CONTRIBUTING.md](./CONTRIBUTING.md).
```

Comment:
nit: `For guide of making contribution please check` -> `Please see [contribution guide](...) for more details`